src/providers/data_provider_be.c

```diff
@@ -112,7 +112,7 @@ static int get_offline_timeout(struct be_ctx *ctx)
 
 void be_mark_offline(struct be_ctx *ctx)
 {
-    int offline_timeout;
+    int offline_timeout, offline_timeout_max;
```

Comment:
This declaration should be split into two separated lines according to convention in this file.

Reply:
Sure, I've corrected this.